### PR TITLE
Use ServiceAddress in the db example in README.md.

### PR DESCRIPTION
**Pull request by markus**
Address is the IP of a Consul node. ServiceAddress is the IP of the service host. https://www.consul.io/api/catalog.html#sample-response-2

### CODE REVIEW

**Comment by johnhamelink** [comment]
@markus would you be able to implement the changes so we can merge this in? 😄

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Here's what a production database.yml file might look like:
 production:
   adapter:            postgresql
   encoding:           unicode
-  host:               <%= Diplomat::Service.get('postgres').Address %>
+  host:               <%= Diplomat::Service.get('postgres').ServiceAddress %>
```
Comment (pierresouchay) [inline README.md]: Most of the time ServiceAddress can be `""`, in that case, we should do:
```
service_instance = Diplomat::Service.get('postgres').ServiceAddress
service_instance.ServiceAddress || service_instance.Address
```
I suspect no ?

**Comment by pierresouchay** [comment]
@markus Still interested with this ? Or Should I close this PR?

**Comment by markus** [comment]
@pierresouchay, I'm closing this one.

**Comment by pierresouchay** [comment]
@markus OK, thank you.

I am gonna try to fix all issues and improve quite a bit the API in next few weeks

Don't hesitate to open issues and open PRs

Regards

**Pull request closed without merging**